docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -0,0 +1,34 @@
+# Restricting S3 access to specific prefix in bucket
+
+> User will still have access to the full lists of S3 objects,
+packages and logical keys inside of packages.
+
+Create "custom" role or policy with these permissions:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
```

Comment:
Do we need to add a principal?

Reply:
No. It can be used only in resource-based policies. 
https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html